linux_os/guide/system/software/sudo/sudo_remove_no_authenticate/ansible/shared.yml

```diff
@@ -4,4 +4,4 @@
 # complexity = low
 # disruption = low
 
-{{{ ansible_sudo_remove_config("!authenticate", "!authenticate") }}}
+{{{ ansible_sudo_remove_config("!authenticate", "\!authenticate") }}}
```

Comment:
Is the backslash needed? The remediation works for me even without it.

Reply:
It should work both ways... I was just following the same regex from existent OVAL:

https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/sudo/sudo_remove_no_authenticate/oval/shared.xml#L37